src/questdb/dataframe.pxi

```diff
@@ -838,9 +846,9 @@ cdef void_int _dataframe_category_series_as_arrow(
             f'Bad column {pandas_col.name!r}: ' +
             'Unsupported arrow category index type. ' +
             f'Got {(<bytes>format).decode("utf-8")!r}.')
-    
+
     format = col.setup.arrow_schema.dictionary.format
-    if strncmp(format, _ARROW_FMT_SML_STR, 1) != 0:
+    if (strncmp(format, _ARROW_FMT_UTF8_STRING, 1) != 0) and (strncmp(format, _ARROW_FMT_LRG_UTF8_STRING, 1) != 0):
```

Comment:
Either ensure there's a test covering categorical types with large string backing, or avoid supporting something we can't cover in tests.